client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/price-summary.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { map, sum } from 'lodash';
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import formatCurrency from 'lib/format-currency';
+import {
+	getPriceBreakdown,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+const PriceSummary = ( { priceBreakdown, translate } ) => {
+	if ( ! priceBreakdown ) {
+		return null;
+	}
+
+	const renderService = ( itemName, itemCost, key, isTotal ) => {
+		const className = classNames( 'label-purchase-modal__price-item', {
+			'label-purchase-modal__price-item-total': isTotal,
+		} );
+		return (
+			<div key={ key } className={ className }>
+				<div className="label-purchase-modal__price-item-name">{ itemName }</div>
+				<div>{ formatCurrency( itemCost, 'USD' ) }</div>
+			</div>
+		);
+	};
+
+	const retailSum = sum( map( priceBreakdown, 'retailRate' ) );
+	const total = sum( map( priceBreakdown, 'rate' ) );
+	const discount = total - retailSum;
+
+	return (
+		<Card>
+			{ priceBreakdown.map( ( service, index ) => renderService( service.title, service.retailRate, index ) ) }
+			{ 0 > discount ? renderService( translate( 'Your discount' ), discount, 'discount' ) : null }
```

Comment:
Hi! I've found a possible matching string that has already been translated **41** times:
`translate( 'Discount' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/19935-label-price-breakdown)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).